src/main/java/gyro/aws/autoscaling/AutoScalingTagFilter.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020, Brightspot.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.aws.autoscaling;
+
+import java.util.List;
+
+import gyro.aws.Copyable;
+import gyro.core.resource.Diffable;
+import gyro.core.validation.Range;
+import gyro.core.validation.Regex;
+import software.amazon.awssdk.services.autoscalingplans.model.TagFilter;
+
+public class AutoScalingTagFilter extends Diffable implements Copyable<TagFilter> {
+
+    private String key;
+    private List<String> values;
+
+    /**
+     * The tag key.
+     */
+    @Regex(value = "[\\u0020-\\uD7FF\\uE000-\\uFFFD\\uD800\\uDC00-\\uDBFF\\uDFFF\\r\\n\\t]*", message = "Alphanumeric characters and symbols excluding basic ASCII control characters.")
```

Comment:
The regex becomes part of a full sentence.  Verify how this looks by triggering the validation.
```suggestion
    @Regex(value = "[\\u0020-\\uD7FF\\uE000-\\uFFFD\\uD800\\uDC00-\\uDBFF\\uDFFF\\r\\n\\t]*", message = "alphanumeric characters and symbols excluding basic ASCII control characters.")
```